config/default.yml

```diff
@@ -662,6 +662,12 @@ Style/SingleLineBlockParams:
 Style/SingleLineMethods:
   AllowIfMethodIsEmpty: true
 
+Style/SpecialGlobalVars:
+  EnforcedStyle: use_verbose_names
```

Comment:
I'd say those are "descriptive" and the others are "cryptic". :-)

Let's rename verbose to descriptive or english. I think verbose carries a different connotation.